README.adoc

```diff
@@ -39,6 +39,22 @@ are supported.
 
 By default we include the URL for link:https://github.com/snowplow/iglu-central[Iglu Central], which includes schemas for the built-in tracking JSONs. You can add as many repositories as you like.
 
+==== Using the universal debugger
+
+The universal debugger uses a link:https://ngrok.com/[Ngrok] tunnel in order to process events. Ngrok provides a HTTP endpoint that we can use as the temporary collector endpoint.
```

Comment:
Maybe specifically call out that this lets you test events from _other devices/platforms_ (maybe mention mobile/server/iot), if they are configured to send events to the tunnel. It's implicitly in `universal` but I think worth calling out a bit more clearly to seed ideas for the use-cases.